tests/cxx/unit_tests/pluginplay/printing/document_modules.cpp

```diff
@@ -52,16 +50,4 @@ TEST_CASE("document_modules") {
                               std::filesystem::directory_iterator{}) ==
                 correct);
     }
-
-    SECTION("Modules Loaded") {
-        // Load your modules
-        pluginplay_examples::load_modules(mm);
-
-        // Generate module documentation at the given path
-        pluginplay::printing::document_modules(mm, docs_path);
-
-        REQUIRE(std::distance(std::filesystem::directory_iterator(docs_path),
-                              std::filesystem::directory_iterator{}) ==
-                mm.size() + 1); // +1 for index.rst
-    }
```

Comment:
Why are we deleting this unit test?

Reply:
Since it depends on `doc_snippets` I moved the test there. But, I can move it back if that makes more sense.

Reply:
Sorry I missed that it was moved. I agree it makes more sense in `doc_snippets`.